client/src/components/profile/components/utils/certification.ts

```diff
@@ -0,0 +1,152 @@
+import { Certification } from '../../../../../../shared/config/certification-settings';
+import { User } from '../../../../redux/prop-types';
+
+export const getCertifications = (user: User) => {
```

Comment:
Is there no way to type this with what we have in `cert-and-project-map.ts` such that we do not accidentally leave out a cert?

Reply:
We can always share types. Maybe in a separate PR?